test/types/createComponent.spec.ts

```diff
@@ -1,11 +1,9 @@
-import { createComponent, createElement as h, ref, SetupContext } from '../../src';
+import { createComponent, createElement as h, ref, SetupContext, PropType } from '../../src';
 import Router from 'vue-router';
 
 const Vue = require('vue/dist/vue.common.js');
 
-type Equal<Left, Right> = (<U>() => U extends Left ? 1 : 0) extends (<U>() => U extends Right
-  ? 1
-  : 0)
+type Equal<Left, Right> = (<U>() => U extends Left ? 1 : 0) extends <U>() => U extends Right ? 1 : 0
```

Comment:
Would you please revert this style change? IMHO the code is more readable with parentheses.

Reply:
@sodatea 

Sure 😄